src/pool-bin/BinPositionManager.sol

```diff
@@ -49,6 +50,7 @@ contract BinPositionManager is
     using BinTokenLibrary for PoolId;
     using BinPoolParametersHelper for bytes32;
     using SlippageCheck for BalanceDelta;
+    using SafeCastTemp for *;
```

Comment:
(optional) wonder if we can do `using SafeCastTemp for uint256` -- just to match the code on CLPositionManager